### PR TITLE
add build and development instructions to README

### PR DESCRIPTION
**Pull request by schmittlauch**
So far I mostly added the build instructions from #46 to the README, so possible contributors could find them easier.
I also modified the dev_symlinks.sh script to conveniently remove the created symlinks again.

@thp: I'd be interested in how your development and build workflow used to be and which tools/ editors (QtCreator?) you used. I haven't really worked with the code so far, but this should be a good basis for getting started.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# gPodder 4 for Sailfish OS
+
+This repository contains the platform-specific customizations and packaging for
+running gPodder 4 on Sailfish OS. Python 3.3.3 and PyOtherSide is already included
+in the device repositories starting with Sailfish OS 1.0.3.8.
+
+## getting started with development
```
Comment (jeena) [inline README.md]: Headlines should start with uppercase.

File: dev_symlinks.sh [hunk 2]
```diff
@@ -1,9 +1,65 @@
...
 # Create symlinks into Git submodules, so the project can
 # be started with qmlscene directly from a source checkout
 
-ln -sf gpodder-core/src/gpodder .
-ln -sf podcastparser/podcastparser.py .
-ln -sf gpodder-ui-qml/main.py .
-ln -sf ../gpodder-ui-qml/common qml
-ln -sf minidb/minidb.py .
+# array of symlinks in form "$TARGET","$LINK_NAME"
+SYMLINKS=("gpodder-core/src/gpodder","gpodder" "podcastparser/podcastparser.py","podcastparser.py" "gpodder-ui-qml/main.py","main.py" "../gpodder-ui-qml/common","qml/common" "minidb/minidb.py","minidb.py")
```
Comment (jeena) [inline dev_symlinks.sh]: I think this array would be easier to maintain if each entry would be in a new line.

Comment (schmittlauch) [reply dev_symlinks.sh]: I have to figure out whether that's possible, because bash arrays are a bit weird. Or I just rewrite the whole script in python.

Comment (jeena) [reply dev_symlinks.sh]: You can always escape new lines with `\` like this:

```bash
SYMLINKS=( \
        "gpodder-core/src/gpodder","gpodder" \
        "podcastparser/podcastparser.py","podcastparser.py" \
        "gpodder-ui-qml/main.py","main.py" \
        "../gpodder-ui-qml/common","qml/common" \
        "minidb/minidb.py","minidb.py" \
)
```

File: README.md (hunk 1 above)
Comment (schmittlauch) [reply README.md]: I'd say *may* here, but I don't really mind changing it.

**Comment by schmittlauch** [comment]
@thp Any chances of getting this merged? I don't think this requires testing from your side, although feedback about your old workflow is welcome.

**Pull request merged**